DEV/org.openl.rules.util/src/org/openl/rules/util/Statistics.java

```diff
@@ -37,6 +37,22 @@ public void processNonNull(T value) {
         });
     }
 
+    public static <T extends Number> Double standardPopulationDeviation(T... values) {
+        Double avg = Avg.avg(values);
+        return process(values, new Result<T, Double>() {
+            @Override
+            public void processNonNull(T value) {
+                double doubleValue = value.doubleValue();
+                result = result == null ? Math.pow((doubleValue-avg), 2) : (result + Math.pow((doubleValue-avg), 2));

```

Comment:
Consider the following simplification (left and right parts are doing the same pow operation):
```suggestion
                Double tmp = Math.pow((doubleValue - avg), 2);
                result = result == null ? tmp : result + tmp;

```